test/src/test/java/org/corfudb/integration/CorfuReplicationUpgradeIT.java

```diff
@@ -1008,4 +875,38 @@ public void testLocalClusterRollingUpgrade() throws Exception {
             sinkReplicationServer.destroy();
         }
     }
+
+    private void verifyRollingUpgrade(boolean source) throws Exception {
+        CorfuStore corfuStore;
+        DefaultAdapterForUpgrade defaultAdapterForUpgrade;
+
+        if (source) {
+            corfuStore = corfuStoreSource;
+            defaultAdapterForUpgrade = new DefaultAdapterForUpgradeSource(sourceRuntime);
+        } else {
+            corfuStore = corfuStoreSink;
+            defaultAdapterForUpgrade = new DefaultAdapterForUpgradeSink(sinkRuntime);
+        }
+
+        corfuStore.openTable(CORFU_SYSTEM_NAMESPACE, REPLICATION_EVENT_TABLE_NAME,
```

Comment:
The fact that the IT needs to open this table indicates that the code won't execute without it - this means in a real deployment it will fail with a `IllegalArgumentException` asking for the `openTable()` with full schema.
I think we should ensure that the tables get opened first right?

Reply:
Addressed in https://github.com/CorfuDB/CorfuDB/pull/3596